marlowe-playground-client/src/MainFrame/State.purs

```diff
@@ -447,6 +462,18 @@ handleAction _ CloseModal = assign _showModal Nothing
 
 handleAction _ (ChangeProjectName name) = assign _projectName name
 
+handleAction settings (OpenLoginPopup intendedAction) = do
+  authRole <- liftAff openLoginPopup
+  handleAction settings CloseModal
+  assign (_authStatus <<< _Success <<< authStatusAuthRole) authRole
+  case authRole of
+    Anonymous -> do
+      liftEffect $ Console.log "User not authenticated, omiting action"
+      pure unit
+    GithubUser -> do
+      liftEffect $ Console.log "User authenticated, executing action"
+      handleAction settings intendedAction
```

Comment:
can get rid of this console logging stuff now, right?

Reply:
done